examples/terraform/gce/README.md

```diff
@@ -0,0 +1,40 @@
+# Terraform
+
+## GCE Provider configuration
+
+### Credentials
+
+Per https://www.terraform.io/docs/providers/google/provider_reference.html#configuration-reference
+ether of the following ENV variables should be accessible:
+* `GOOGLE_CREDENTIALS`
```

Comment:
Can we add those variables to [the `environment_variables.md` document](https://github.com/kubermatic/kubeone/blob/master/docs/environment_variables.md)?

Reply:
👍